### PR TITLE
New Power Metrics and Power Management capabilities on StarlingX

### PR DESCRIPTION
**Pull request by dadafros**
Add a blogpost introducing new features on StarlingX

### CODE REVIEW

File: src/pages/blog/starlingx-configurable-power-management.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+# Conclusion
+
+In conclusion, with the ability to collect and publish power data and metrics, StarlingX takes a significant step forward in providing users with valuable insights into their energy usage patterns, contributing to more informed decision-making. The introduction of the Kubernetes Power Manager feature enables the system to optimize power usage according to user needs and workload characteristics. This feature offers various capabilities, including the ability to assign power profiles on a per-core or per-application basis, control frequency ranges for individual cores, govern core behavior within these ranges, specify which power levels (C-States) a particular core can access, and determine system behavior in response to workloads with known intervals or demands. Throughout this blog post, we've explored common use cases where the Kubernetes Power Manager can be advantageous, along with considerations regarding its operation and impacts on the platform.
```
Comment (ildikov) [inline src/pages/blog/starlingx-configurable-power-management.md:154]:
```suggestion
Throughout this blog post, we've explored common use cases where the Kubernetes Power Manager can be advantageous, along with considerations regarding its operation and impacts on the platform.

In conclusion, with the ability to collect and publish power data and metrics, StarlingX takes a significant step forward in providing users with valuable insights into their energy usage patterns, contributing to more informed decision making. The introduction of the Kubernetes Power Manager feature enables the system to optimize power usage according to user needs and workload characteristics. This feature offers various capabilities, including the ability to assign power profiles on a per-core or per-application basis, control frequency ranges for individual cores, govern core behavior within these ranges, specify which power levels (C-States) a particular core can access, and determine system behavior in response to workloads with known intervals or demands.
```

File: src/pages/blog/starlingx-configurable-power-management.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+For further information about the features, please refer to the documentation provided below:
```
Comment (ildikov) [inline src/pages/blog/starlingx-configurable-power-management.md:156]:
```suggestion
For further information about the features, check out the project documentation:
```

File: src/pages/blog/starlingx-configurable-power-management.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
+---
+templateKey: blog-post
+title: New Power Metrics and Power Management capabilities on StarlingX
+author: Davi Frossard
+date: 2023-11-16T11:14:00.000Z
```
Comment (ildikov) [inline src/pages/blog/starlingx-configurable-power-management.md:5]: The StarlingX release is currently scheduled to come out in March 2024. Would it be ok with you to schedule this blog post around the time of the release?

**Review by ildikov — COMMENTED** [review]
Hi Davi,

Thank you for contributing this blog post, I think it looks great overall! I added a few minor suggestions in line.

I also added a comment related to scheduling the blog post, please let me know what you think.

Best Regards,
Ildikó

File: src/pages/blog/starlingx-configurable-power-management.md (hunk 3 above)
Comment (dadafros) [reply src/pages/blog/starlingx-configurable-power-management.md:5]: Yes, sounds good!

File: src/pages/blog/starlingx-configurable-power-management.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+  - Power Config Controller:  part of the power manager, is responsible for evaluating the presence of a default power configuration for the node and, when present, starting the Power Node Agent.
+  - Power Config: describes the power configuration of a given node indicating one or more profiles that can be used.
+
+- Power Node Agent:  per-node pod managed by a DaemonSet, responsible for managing the power profiles applied. Communicates with Power Manager to establish power policies to be applied to the node.
+
+  - Power Profile: establishes CPU operating frequency ranges, Energy Performance Preference (EPP), and governor. The profile has a generic aspect, that is, it only describes a possible style of power control, and its application is the responsibility of Power Workload. The pods' deployment files, or Power Config files, can indicate the profiles they want to use by including the device power.intel.com/<POWERPROFILE>. It is also important to note that all CPUs not assigned to a specific power profile are pooled in a profile known as the *Shared*. This profile must be created manually by the user.
```
[the remark this answers is not in the document]
Comment (dadafros) [reply src/pages/blog/starlingx-configurable-power-management.md:59]: I will mention that in the introduction.

**Comment by dadafros** [comment]
Hi @ildikov can we proceed with publication?
Thank you!

**Comment by ildikov** [comment]
Hi Davi,

Yes, I will publish it early next week.

It's a common practice to spread out content, that way we have periodic opportunities to point people to something new and spread out social media and other promo about the project and the 9.0 release. It's much harder to do if all the content is released at the same time. That is why this one haven't gone out yet.

Best Regards,
Ildikó

**Pull request merged**